### PR TITLE
Use Github Actions instead of travis CI

### PR DESCRIPTION
**Pull request by bytehead**

### CODE REVIEW

**Comment by sheeep** [comment]
# [Codecov](https://codecov.io/gh/contao/phpstan/pull/12?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@07ab15c`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/contao/phpstan/pull/12?src=pr&el=tree)

```diff
@@            Coverage Diff            @@
##             master      #12   +/-   ##
=========================================
  Coverage          ?   23.12%           
  Complexity        ?       74           
=========================================
  Files             ?        9           
  Lines             ?      147           
  Branches          ?        0           
=========================================
  Hits              ?       34           
  Misses            ?      113           
  Partials          ?        0           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/contao/phpstan/pull/12?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/contao/phpstan/pull/12?src=pr&el=footer). Last update [07ab15c...8ce697d](https://codecov.io/gh/contao/phpstan/pull/12?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by sheeep** [comment]
Que?

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018 1up GmbH
+Copyright (c) 2020 1up GmbH
```
Comment (leofeyer) [inline LICENSE]: Shouldn't this be `2018–2020`? Or maybe just `Copyright (c) 1up GmbH` so you do not have to update the file every year.

File: composer.json [hunk 2]
```diff
@@ -52,6 +53,22 @@
...
         }
     },
+    "scripts": {
+        "all": [
+            "@unit-tests",
+            "@cs-fixer",
+            "@phpstan"
+        ],
+        "cs-fixer": [
+            "vendor/bin/php-cs-fixer fix src -vvv --dry-run --ansi"
```
Comment (leofeyer) [inline composer.json]:
```suggestion
            "vendor/bin/php-cs-fixer fix src --diff --diff-format=udiff --dry-run --ansi"
```

File: LICENSE (hunk 1 above)
Comment (bytehead) [reply LICENSE]: Removed in 1724c2f.

**Review by leofeyer — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (bytehead) [reply composer.json]: Adapted in 0036c3c.

**Review by leofeyer — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                  coverage: none
+
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: Install the dependencies
+              run: composer install --no-interaction --no-suggest
+
+            - name: Check the coding style
+              run: vendor/bin/php-cs-fixer fix src --dry-run --ansi
```
Comment (leofeyer) [inline .github/workflows/ci.yml]:
```suggestion
              run: vendor/bin/php-cs-fixer fix src --diff --diff-format=udiff --dry-run --ansi
```

**Review by leofeyer — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 3 above)
Comment (bytehead) [reply .github/workflows/ci.yml]: Adapted in b21a2a9.

**Review by leofeyer — APPROVED** [review]

**Pull request merged**